lib/libzutil/zutil_import.c

```diff
@@ -929,22 +1006,43 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 
 	if (lio_listio(LIO_WAIT, aiocbps, VDEV_LABELS, NULL) != 0) {
 		int saved_errno = errno;
+		boolean_t do_slow = B_FALSE;
+		error = -1;
 
 		if (errno == EAGAIN || errno == EINTR || errno == EIO) {
 			/*
-			 * A portion of the requests may have been submitted.
-			 * Clean them up.
+			 * A portion of the requests may have
+			 * been submitted. Clean them up.
 			 */
 			for (l = 0; l < VDEV_LABELS; l++) {
 				errno = 0;
-				int r = aio_error(&aiocbs[l]);
-				if (r != EINVAL)
-					(void) aio_return(&aiocbs[l]);
+				switch (aio_error(&aiocbs[l])) {
+					case EINVAL:
```

Comment:
The case labels can be at the same indent level as the switch.

Reply:
Works for me. Done.